server/sonar-web/src/main/js/apps/projects/filters/LanguageFilterOption.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import { connect } from 'react-redux';
+import React from 'react';
+
+class LanguageFilterOption extends React.Component {
+  render () {
+    const languageName = this.props.language ? this.props.language.name : this.props.languageKey;
+    return (
+      <span>{languageName}</span>
+    );
+  }
+}
+
+const mapStateToProps = (state, ownProps) => ({
+  language: state.languages[ownProps.languageKey]
```

Comment:
yeah, I think this is a typo that there is this {{key}} parameter, you can clean it up